src/components/ProductCard.tsx

```diff
@@ -82,8 +82,15 @@ export default function ProductCard({ content }: { content: Content }) {
         </div>
 
         <div className="flex flex-col w-[413.33px] h-[508px] items-center relative">
-          <div className="top-10 relative">
-            <Image src={cardImage} height={300} width={300} alt="cardImage" />
+          <div className="pt-10 absolute">
+            <Image
+              src={cardImage}
+              height={300}
+              width={300}
+              alt="cardImage"
+              className="text-center"
+              style={{ height: 250, width: 300 }}
```

Comment:
heights are different

Reply:
Also, there has to be a different way to do this, seems odd you would have to declare it again in style. You mind looking into this?

Reply:
removing styling, only using width and height properties.